src/main/java/uk/gov/hmcts/darts/cache/token/service/Token.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.darts.cache.token;
+package uk.gov.hmcts.darts.cache.token.service;
```

Comment:
Are you able to have a look at some of the Issues / Code Smells in sonar?
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=242&id=uk.gov.hmcts.reform%3Adarts-gateway

Reply:
yep done and corrected

Reply:
Still showing 32 issues - if you can have another look at the filters by severity, Medium for example?

Reply:
Was relying on the gate criteria of the build to guide me. I will take a look